dev/MRTCore/packaging/Microsoft.ApplicationModel.Resources.PriGen.targets

```diff
@@ -205,8 +205,6 @@
     <PriInitialPath Condition="'$(PrependPriInitialPath)' == 'true' and '$(AppxPackage)' == 'true' and '$(PriInitialPath)' == ''"></PriInitialPath>
     <PriInitialPath Condition="'$(PrependPriInitialPath)' == 'true' and '$(AppxPackage)' != 'true' and '$(PriInitialPath)' == ''">$(TargetName)</PriInitialPath>
     <ProjectPriFileName Condition="'$(AppxPackage)' == 'true' and '$(ProjectPriFileName)' == ''">resources.pri</ProjectPriFileName>
-    <!-- An empty '$(ApplicationType)' and '$(WindowsAppContainer)' being false corresponds to a C++, Win32 app. -->
-    <ProjectPriFileName Condition="'$(ApplicationType)' == '' and '$(WindowsAppContainer)' == 'false' and '$(ProjectPriFileName)' == ''">resources.pri</ProjectPriFileName>
```

Comment:
><ProjectPriFileName Condition="'$(ApplicationType)' == '' and '$(WindowsAppContainer)' == 'false' and '$(ProjectPriFileName)' == ''">resources.pri</ProjectPriFileName> [](start = 4, length = 167)

what makes packaged WinUI app not satisfying this condition?

Reply:
I asked devdiv folks. I was told that this is one way to check for win32. Looking in the msbuild.binlog file for a WinUI3 .vcxproj:
winui_desktop_packaged_app_cpp.vcxproj id:4
    Property ApplicationType = Windows Store
Looking in the msbuild.binlog file for a WinUI3 .csproj, this property isn't defined.

Reply:
we need to find a way to get it back to resources.pri for unpackaged win32 app later.

---
In reply to: [587970850](https://github.com/microsoft/ProjectReunion/pull/526#discussion_r587970850) [](ancestors = 587970850)